circle.yml

```diff
@@ -4,8 +4,7 @@ machine:
   node:
     version: v5.5.0
   environment:
-    NODE_012_VERSION: v0.12.0
-    NODE_4_VERSION: v4.2.2
+    NODE_4_VERSION: v4.5.0
```

Comment:
Yeah... I mean parallelization might be handy if we want to test node distributions. But now... It might not be handy other than to ensure all contributors would be able to work with it.